fmcapi/fmc.py

```diff
@@ -281,7 +281,7 @@ def send_to_api(
             self.error_response = json_response
             if response:
                 response.close()
-            return None
+            return json_response  #allows me to use the response in my script
```

Comment:
This is a massively backward compatibility breaking change. Anyone who is expecting a None (or a non-truthy response) will now get the response. So in their code if they detect a failure with a None they will be assume the API was successful. You can see three lines above that the error is stored in self.error_response. So what you script should be doing is if the returned response is None then you should check your FMC Object for the error_response attribute to get the data you need. 

This change will have to be removed before approve this pull request.